pip/req/req_install.py

```diff
@@ -460,11 +458,9 @@ def run_egg_info(self):
     # FIXME: This is a lame hack, entirely for PasteScript which has
     # a self-provided entry point that causes this awkwardness
     _run_setup_py = """
```

Comment:
I think this "lame hack" present since c2000d7de68ef (7 years now) could now be removed ?